### ISSUE TITLE
Can't scroll commandList inside a Dialog using mouse wheel

### ISSUE THREAD

**Issue by Rrhapsod** [pacocoursey/cmdk#272]
Hi, I have a Combobox from shadcn inside a Dialog component. It works perfectly, but I can't scroll the itens list with mouse wheel, I can only scroll by dragging the scroll bar.

Testing the same Combobox outside the shadcn Dialog, I can scroll perfectly with mouse and dragging the scroll bar.

Maybe the problem is with focus?

Structure:

```javascript

<Dialog open={isOpen} onOpenChange={onOpenChange}>
      <DialogContent>
            <div>
              <DialogHeader>
                <DialogTitle>
                </DialogTitle>
                <DialogDescription>
                </DialogDescription>
              </DialogHeader>
              <div className="my-4 flex w-full flex-col items-center gap-4">
                  <div className="flex flex-col justify-around gap-6 md:flex-row">
                    <div className="flex flex-col items-center gap-2">
                      <Label>Selecione uma Conta Contábil:</Label>
                               <Popover open={open} onOpenChange={setOpen}>
                                <PopoverTrigger asChild>
                                  <Button
                                    variant="sallamos"
                                    role="combobox"
                                    aria-expanded={open}
                                  >
                                  <ChevronsUpDown/>
                                </Button>
                              </PopoverTrigger>
                              <PopoverContent className="w-64 p-0">
                                <Command
                                  className="bg-slate-800 text-white"
                                  value={searchText}
                                  onValueChange={handleSearch}
                                  shouldFilter={true}
                                  filter={(value, search) =>
                                    props.lista
                                      .find((conta) => conta.id === value)
                                      ?.contaContabil.toLowerCase()
                                      .includes(search.toLowerCase())
                                      ? 1
                                      : 0
                                  }
                                >
                                  <CommandInput
                                    placeholder="Buscar..."
                                  />
                                  <CommandList>
                                    <CommandEmpty>Conta Contábil não encontrada.</CommandEmpty>
                                    <CommandGroup className="outline-none focus:border-slate-100 focus:outline-none">
                                      {props.lista.map((conta) => (
                                        <CommandItem
                                          className={`cursor-pointer aria-selected:bg-slate-200 ${
                                            conta.disabled ? "text-gray-500" : "text-slate-50"
                                          }`}
                                          key={conta.id}
                                          value={conta.id}
                                          disabled={conta.disabled}
                                          onSelect={(currentValue) => {
                                            setValue(currentValue === value ? "" : currentValue);
                                            setOpen(false);
                                            props.setContaId(
                                              currentValue === value ? "" : currentValue,
                                            );
                                          }}
                                        >
                                          <Check
                                            className={cn(
                                              "mr-2 h-4 w-4 shrink-0",
                                              value === conta.id ? "opacity-100" : "opacity-0",
                                            )}
                                          />
                                          {`${conta.codArvore} - ${conta.contaContabil}`}
                                        </CommandItem>
                                      ))}
                                    </CommandGroup>
                                  </CommandList>
                                </Command>
                              </PopoverContent>
                            </Popover>
                      <Button variant="secondary">Incluir Conta</Button>
                    </div>
                  </div>
              </div>
              <DialogFooter>
              </DialogFooter>
      </DialogContent>
    </Dialog>

```

**Comment by Rrhapsod** [comment]
So, I fixed set modal on Popover to true.

```javascript

<Popover open={open} onOpenChange={setOpen} modal={true}>

```

**Comment by 6jonathanaloi6** [comment]
Thankl

**Comment by ggimenes** [comment]
I did a dirty hack here:

```
useEffect(() => {
    if (!open) return;

    let scrollElement: HTMLElement | null = null;
    const cleanup = () => {
      scrollElement?.removeEventListener('wheel', handleWheel);
    };

    const handleWheel = (e: WheelEvent) => {
      if (!scrollElement) return;
      const delta = e.deltaY;
      scrollElement.scrollTop += delta;
      e.preventDefault();
    };

    const timer = setTimeout(() => {
      scrollElement = document.querySelector('[data-scrollable]');
      if (scrollElement) {
        scrollElement.addEventListener('wheel', handleWheel, { passive: false });
      }
    }, 100);

    return () => {
      cleanup();
      clearTimeout(timer);
    };
  }, [open]);
```

And then:
```
<CommandList className="max-h-[300px] overflow-y-auto" data-scrollable>
```

**Comment by if1eight0sty** [comment]
@Rrhapsod it did not worked for me. this is what i have done. what am I doing wrong?

```js
<Dialog open={isDialogOpen} onOpenChange={setIsDialogOpen}>
      <DialogContent>
             <Popover
                  open={isPopoverOpen}
                  onOpenChange={setIsPopoverOpen}
                  modal={modalPopover}
                >
                    <PopoverTrigger ref={ref} asChild>
                        <Button
                            ref={buttonRef}
                            {...props}
                            onClick={handleTogglePopover}
                       >
                        </Button>
                    </PopoverTrigger>
                    <PopoverContent
                        className="min-w-[var(--radix-popper-anchor-width)] p-0"
                        align="start"
                        onEscapeKeyDown={() => setIsPopoverOpen(false)}
                    >
                        <Command>
                            <CommandInput
                                placeholder="Search..."
                                onKeyDown={handleInputKeyDown}
                            />
                            <CommandList className='max-h-[300px] overflow-y-auto'>
                                <CommandEmpty>No results found.</CommandEmpty>
                                <CommandGroup>
                                    <CommandItem
                                        key="all"
                                        className="cursor-pointer"
                                      >
                                         <CheckIcon className="w-4 h-4" />
                                        <span>Select All</span>
                                    </CommandItem>
```

**Comment by devodii** [comment]
> I did a dirty hack here:
> 
> ```
> useEffect(() => {
>     if (!open) return;
> 
>     let scrollElement: HTMLElement | null = null;
>     const cleanup = () => {
>       scrollElement?.removeEventListener('wheel', handleWheel);
>     };
> 
>     const handleWheel = (e: WheelEvent) => {
>       if (!scrollElement) return;
>       const delta = e.deltaY;
>       scrollElement.scrollTop += delta;
>       e.preventDefault();
>     };
> 
>     const timer = setTimeout(() => {
>       scrollElement = document.querySelector('[data-scrollable]');
>       if (scrollElement) {
>         scrollElement.addEventListener('wheel', handleWheel, { passive: false });
>       }
>     }, 100);
> 
>     return () => {
>       cleanup();
>       clearTimeout(timer);
>     };
>   }, [open]);
> ```
> 
> And then:
> 
> ```
> <CommandList className="max-h-[300px] overflow-y-auto" data-scrollable>
> ```

TLDR;
```typescript
  <PopoverContent
      onWheel={(e) => e.stopPropagation()}
     />
```
